api/server.go

```diff
@@ -35,7 +35,7 @@ type rpcHandler struct {
 	server *rpc.Server
 }
 
-type httpServer struct {
```

Comment:
just renamed to server, because it's not HTTP only since it allows websockets and also the package is API so it makes it `api.Server` which makes more sense.